fv3_cap.F90

```diff
@@ -222,6 +222,16 @@ subroutine InitializeAdvertise(gcomp, rc)
     write(msgString,'(A,l6)') trim(subname)//' cplprint_flag = ',cplprint_flag
     call ESMF_LogWrite(trim(msgString), ESMF_LOGMSG_INFO, rc=rc)
 
+    call ESMF_AttributeGet(gcomp, name="Diagnostic", value=value, defaultValue="0", &
```

Comment:
What is this diagnostic variable used for? Do we need it for all the applications?

Reply:
I pulled the Diagnostic value from the FV3 component attributes and store it for later use when I want to write out the grid information or not. I can remove the diagnostics that I added and used for developing the coupled regional grid. Feel free to make suggestions.

Reply:
This comment has been addressed in https://github.com/NOAA-EMC/fv3atm/pull/355/commits/6ac48b1d36b5559500dcb4718bae9140206fc2b8. Diagnostic was removed.